spacemesh/v2alpha1/layer.proto

```diff
@@ -24,6 +25,7 @@ message LayerRequest {
   uint32 end_layer = 2;
   uint64 offset = 3; // adjusts the starting point for data
   uint64 limit = 4; // specifies max number of items to fetch
+  optional SortOrder sort_order = 5; // specifies the sort order (default is ASC)
```

Comment:
Do you need the `optional` keyword if the default is the 0-value enum variant anyway (ASC)? Btw, I'd expect that the default (when not set) is to not sort at all. Wdyt?